Makefile

```diff
@@ -149,9 +149,9 @@ test-e2e: ##@tests Run e2e tests
 	build/env.sh go test -timeout 5m ./e2e/node/...
 	build/env.sh go test -timeout 15m ./e2e/jail/...
 	build/env.sh go test -timeout 20m ./e2e/rpc/...
-	build/env.sh go test -timeout 10m ./e2e/whisper/...
+	build/env.sh go test -timeout 20m ./e2e/whisper/...
```

Comment:
Have you faced an issue with that?

Reply:
During tests, but yes, may be gone now with `EnsureNodeSync()`. So I'll change it back to 10 minutes and test again.